crates/cargo-gpu/src/install.rs

```diff
@@ -231,20 +249,31 @@ impl Install {
 
             build_command.args([
                 "--features",
-                &Self::get_required_spirv_builder_version(spirv_version.date)?,
+                &Self::get_required_spirv_builder_version(spirv_version.date)
+                    .context("getting required spirv builder version")?,
             ]);
 
             log::debug!("building artifacts with `{:?}`", build_command);
 
-            let build_output = build_command
+            build_command
                 .stdout(std::process::Stdio::inherit())
                 .stderr(std::process::Stdio::inherit())
-                .output()?;
-            anyhow::ensure!(build_output.status.success(), "...build error!");
+                .output()
+                .context("getting command output")
+                .and_then(|output| {
+                    log::trace!("STDOUT:\n{}", String::from_utf8_lossy(&output.stdout));
+                    log::trace!("STDERR:\n{}", String::from_utf8_lossy(&output.stderr));
```

Comment:
These lines shouldn't be necessary, since the command inherits stdin and stdout, right? Maybe I'm missing something though.